app/forms/hyrax/forms/collection_form.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OVERRIDE FILE from Hyrax v2.9.0
```

Comment:
- correct format
- make sure override upgrade to Hyrax 3.4
- add reason

Reply:
this one should be a decorator and not an override. still needs a Hyrax 3.4 check